sources/Context.cpp

```diff
@@ -151,15 +151,15 @@ Context::exportOutputs() {
   // create specific par
   file::path parOutput(config_.outputDir());
   parOutput.append(basename_ + ".par");
-  outputs::Par parWriter(outputs::Par::ParDefinition(basename_, parOutput.generic_string(), generators_));
+  outputs::Par parWriter(outputs::Par::ParDefinition(basename_, config_.outputDir(), parOutput.generic_string(), generators_));
```

Comment:
Here I have the impression we pass twice an information, config_.outputDir() and  parOutput.generic_string().
Indeed, parOutput.genereic_string() is built just above from config_.outputDir()

Reply:
technicay correct, as parOutput = config_.outputDir() + basename_. I keep all of these to keep the spirit of putting the filename in the writer definition and to have flexibility in the future if, for one reason or another we would put the diagram file in another directory than the dyd and par. Should I restrict the information ?

Reply:
I don't have any preference, both point of views are OK to me.... (sorry for not helping ;))
Marco I'll let you have the final word on this.

Reply:
ok, let's keep some room for flexibility :)